src/schema/users.ts

```diff
@@ -1636,6 +1709,75 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
 
       return { _: true };
     },
+    addUserCompany: async (
```

Comment:
Does this check if the email already exist?

Reply:
Ah good one, will need to add a test for it, it's a unique index so think the query will already fail on it.

Reply:
If the email exists, but claimed by same user and not verified, we should just "ignore" that it exists and just update verification code
In the case when the user stops midway in the verification process (i.e. computer dies or something) and before the cleanup job gets it. [Thread](https://dailydotdev.slack.com/archives/C07EA1E1XME/p1723730384460159?thread_ts=1723727031.217919&cid=C07EA1E1XME)

Reply:
So resubmitting email will just generate a new code?
I'll add that :)